src/routes/component.test.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { MemoryRouter } from 'react-router-dom';
+import { shallow } from 'enzyme';
+
+import LandingPage from '../pages/landing-page';
+
+import NavigationRoutes from './component';
+import { LANDING_PAGE } from './urls';
+
+describe('<NavigationRoutes/>', () => {
+  it('checks if route navigates to landing page', () => {
+    const wrapper = shallow(
+      <MemoryRouter initialEntries={[LANDING_PAGE]}>
+        <NavigationRoutes />
+      </MemoryRouter>
+    );
+    expect(wrapper.find(LandingPage)).toHaveLength(0);
```

Comment:
This doesn't look like a correct test. We can rather test that each route exists.